app/services/early_career_teachers/create.rb

```diff
@@ -24,12 +24,12 @@ def call
           ParticipantProfileState.create!(participant_profile: profile)
 
           unless year_2020
-            ParticipantMailer.participant_added(participant_profile: profile).deliver_later
+            # ParticipantMailer.participant_added(participant_profile: profile).deliver_later
```

Comment:
in a later PR we will need to uncomment those jobs. They spam sidekiq and and review app server with jobs that will fail because the transaction has not yet been comnited.
Will reinstate later when m2-release branch is ready to be merged

Reply:
we probably want to uncomment before we merge. if you want to disable these can we not wrap around a feature toggle?

Reply:
https://github.com/DFE-Digital/early-careers-framework/pull/1695#discussion_r800554240